templates/resolv.conf.tmpl

```diff
@@ -23,8 +23,5 @@ sortlist {% for sort in sortlist %}{{sort}} {% endfor %}
 {% endif %}
 {% if options or flags %}
 
-options {% for flag in flags %}{{flag}} {% endfor %}
-{% for key, value in options.items() -%}
```

Comment:
or if we started this line with 

```python
{%- for %}
```

Reply:
Pushed a new commit to improve readability.
Does it feel better?
Thanks.